ambry-clustermap/src/main/java/com/github/ambry/clustermap/HelixClusterManager.java

```diff
@@ -559,81 +545,74 @@ public String getLocalDcZkConnectString() {
   }
 
   /**
-   * A callback class for {@link HelixClusterChangeHandler} in each dc to update cluster-wide info (i.e partition-to-replica
-   * mapping, cluster-wide capacity)
+   * Add partition if it's not present in cluster-wide partition map and also update cluster-wide allocated usable
+   * capacity. If the partition already exists, skip addition and return current partition.
+   * @param partition the {@link AmbryPartition} to add (if not present)
+   * @param capacityBytes the capacity of partition in bytes
+   * @return the current {@link AmbryPartition} present in the map.
    */
-  class ClusterChangeHandlerCallback {
```

Comment:
ClusterChangeHandlerCallback was used previously for DynamicClusterChangeHandler to update cluster information back in HelixClusterManager. Since DynamicClusterChangeHandler is an inner class now of HelixClusterManager and has access to its data structures, removed this class.